src/Card/Card.jsx

```diff
@@ -1,25 +1,70 @@
-import './style';
-import React from 'react';
-import Popover from '../Popover/Popover';
-
-export default function Card({ newModal, newDeleteModal, newMovieInfo, movie }) {
-    const ref = React.useRef();
-
-    const handleCardClick = () => {
-        newMovieInfo(movie);
-    }
-
-    return (
-        <>
-            <div className='card' ref={ref}>
-                <img src={movie.url} onClick={handleCardClick} id={movie.id} />
-                <Popover newModal={newModal} newDeleteModal={newDeleteModal} />
-                <div className='description'>
-                    <div className='year'>{movie.year}</div>
-                    <p className='title'>{movie.name}</p>
-                    <p className='genre'>{movie.genre}</p>
-                </div>
-            </div>
-        </>
-    )
-}
\ No newline at end of file
+import PropTypes from 'prop-types';
+import React from 'react';
+import Popover from '../Popover/Popover';
+
+import './style.css';
+
+function Card({
+  onMovieModalOpen, onDeleteModalOpen, onMovieInfoOpen, movie,
+}) {
+  const handleCardClick = () => {
+    onMovieInfoOpen(movie);
+  };
+
+  return (
+    <div
+      className="card"
+      ref={React.createRef()}

```

Comment:
You don't use this ref, so it can be removed.

Reply:
Removed.